src/jitter.rs

```diff
@@ -422,6 +423,8 @@ impl JitterRng {
     }
 
     fn gen_entropy(&mut self) -> u64 {
+        trace!("JitterRng: generating entropy");
```

Comment:
This gets called a lot when seeding `StdRng` — 256 times. @pitdicker and I already noted that initialising ISAAC via `JitterRng` can be slow; should we fix? Not sure whether it's better to adjust the `seed_from_rng` implementation or to try to make a special case for ISAAC seeded from JitterRng.

Reply:
`collecting entropy` sounds better to me than `generating entropy`.